src/utils.cpp

```diff
@@ -195,6 +196,104 @@ bool is_lit(const Term & l, const Sort & boolsort)
   return false;
 }
 
+
+
+std::string cnf_to_dimacs(Term cnf){
+   TermVec vecs({cnf});
+   TermVec vecs2;
```

Comment:
Please run `clang-format` on the code. It will make it look nicer, e.g., with spaces around operators etc.